### PR TITLE
Dockerize our cc-selfplay binary

### PR DESCRIPTION
**Pull request by amj**
Adds a Dockerfile for our cc selfplay job, and a cc_wrapper.sh to invoke it appropriately.

### CODE REVIEW

File: cluster/selfplay/Makefile [hunk 1]
```diff
@@ -31,6 +31,16 @@ cpu-image-clean:
 		rm staging/requirements.txt
 		rm staging/player_wrapper.sh
 
+cc-image:
+		mkdir -p staging/cc
+		cp ../../requirements.txt staging/
+		cp ../../WORKSPACE staging/
+		cp cc_wrapper.sh staging/
+		cp -r ../../cc/* staging/cc/
```
Comment (sethtroisi) [inline cluster/selfplay/Makefile]: these can be joined to a smaller number of lines if you want

cp ../../requirements.txt ../../WORKSPACE cc_wrapper.sh staging/

File: cluster/selfplay/Makefile [hunk 2]
```diff
@@ -31,6 +31,16 @@ cpu-image-clean:
...
+	    docker build -f cc-Dockerfile -t "gcr.io/$(PROJECT)/$(CC_PLAYER_CONTAINER):$(VERSION_TAG)" .
```
Comment (sethtroisi) [inline cluster/selfplay/Makefile]: one tab + spaces

File: cluster/selfplay/cc-Dockerfile [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV BAZEL_VERSION 0.11.0
+
+WORKDIR /
+RUN mkdir /bazel && \
+    cd /bazel && \
+    curl -H "User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36" -fSsL -O https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
```
Comment (sethtroisi) [inline cluster/selfplay/cc-Dockerfile]: do you really need to set a useragent?

File: cluster/selfplay/cc_wrapper.sh [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+MODEL_NAME=gsutil ls "gs://$BUCKET_NAME/models/*.pb" | sort | tail -n 1
+GAMES=`gsutil ls "gs://$BUCKET_NAME/data/selfplay/$MODEL_NAME/*.zz" | wc -l`
+
+if [ $GAMES -lt 15000 ];
+then
+  bazel-bin/cc/main \
+    --model=$MODEL_NAME \
+    --num_readouts=800 \
+    --mode=selfplay \
+    --resign-threshold=0.90
```
Comment (sethtroisi) [inline cluster/selfplay/cc_wrapper.sh]: parametarized with default

resign_threshold=${0:-0.90}
readouts=${1:-800}

**Comment by Kashomon** [comment]
little bit late, but LGTM ;)

**Pull request merged**